core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer-utils.ts

```diff
@@ -410,6 +410,30 @@ export class BaseUser {
   getCurrentUrlWithoutParameters(): string {
     return this.page.url().split('?')[0];
   }
+
+  /**
+   * This function Waits for a specific text to appear in the body of the page.
+   * This function only waits for rendered text. It does not wait for text in iframes or text that is hidden or not rendered for some reason.
+   * The text matching is case-sensitive and space-sensitive, so it might not find the text if there are differences in case or white space.
+   * @param {string} text - The text to wait for.
+   * @param {number} [timeout=30000] - The maximum time to wait in milliseconds. Defaults to 30000 (30 seconds).
+   */
+  async waitForText(text: string, timeout: number = 30000): Promise<void> {
```

Comment:
waitForTextToAppearInPageBody

Change timeout to timeoutMsec -- always include units.